### PR TITLE
Add the possibility to use the HumanControlBoard with just kineamtics or dynamics

### PR DESCRIPTION
**Pull request by lrapetti**
Currently, the `HumanControlBoard`, which is the device that exposes the output of the HDE as a robot controlboard, is working only if both `HumanStateProvider` and `HumanDynamicsEstimator` are running.

With this PR, I add the possibility of running it, even with just one of the two. This is possible by changing the [network elements](https://github.com/robotology/human-dynamics-estimation/blob/master/conf/xml/Human.xml#L291-L292). To the desired one. Data related to the other device are fixed to be zero.

I have tested the device in combination with the [`WB-toolbox`](https://github.com/robotology/wb-toolbox), after the changes and it is working fine.

### CODE REVIEW

**Comment by lrapetti** [comment]
In the PR I am also removing a left-over deprecated device (`RobotPositionController-old.cpp`)

File: devices/HumanControlBoard/HumanControlBoard.cpp [hunk 1]
```diff
@@ -132,36 +134,60 @@ bool HumanControlBoard::open(yarp::os::Searchable& config)
...
+    {
+        yError() << LogPrefix << "No interface attached to the device.";
+        askToStop();
+        return;
+    }
 
-    std::vector<double> jointPositionsInterface = pImpl->iHumanState->getJointPositions();
-    std::vector<double> jointVelocitiesInterface = pImpl->iHumanState->getJointVelocities();
+    // Initialize buffers for measurements
+    std::vector<double> jointPositionsInterface(pImpl->nJoints, 0.0);
```
Comment (Yeshasvitvs) [inline devices/HumanControlBoard/HumanControlBoard.cpp:161]: Minor: The variable ending of `Interface` is a bit misleading. May be we can drop it as : `jointPositionsInterface` -------> `jointPositions`

Also for other variables

Comment (Yeshasvitvs) [reply devices/HumanControlBoard/HumanControlBoard.cpp:161]: or a suffix as `Data`

**Review by Yeshasvitvs — APPROVED** [review]
Thanks @lrapetti LGTM. Just a minor comment about variable name.
It would also be nice if there is any video or screenshot for future reference

File: devices/HumanControlBoard/HumanControlBoard.cpp (hunk 1 above)
Comment (lrapetti) [reply devices/HumanControlBoard/HumanControlBoard.cpp]: Yes, I was also not very convinced by that variable naming. I have changed to `jointXXXData`

**Comment by Yeshasvitvs** [comment]
Thanks @lrapetti 
Merged.

**Pull request merged**